tests/run.rs

```diff
@@ -104,6 +104,23 @@ fn can_specify_interpreter() -> R<()> {
     Ok(())
 }
 
+#[test]
+fn can_specify_regex() -> R<()> {
+    test_run(
+        r"
+            |#!/usr/bin/env bash
+            |cp foo bar
+        ",
+        r#"
+            |protocols:
+            |  - protocol:
+            |    - regex: cp foo \w+
+        "#,
+        Ok(()),
+    )?;
+    Ok(())
+}
```

Comment:
Should we add one more test that shows that regexes can be used to match on the command, not just the arguments?